hack/observability/cluster-api-visualizer/chart/templates/clusterrole.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: capi-visualizer
+rules:
+# TODO: Consider restricting the rights to the relevant CAPI apiGroups.
```

Comment:
Can we open a good-first-issue for this?

Reply:
Sure thing!